src/tooltip/test/tooltip-template.spec.js

```diff
@@ -69,11 +69,10 @@ describe('tooltip template', function() {
     scope.templateUrl = '';
     scope.$digest();
 
+    $timeout.flush();
```

Comment:
Yes, the test needs the timeout to flush properly in order to enable the correct state.